skel/problem/generators/example.py

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/python3
+import sys
+import random
+
+#Init seed with first argument
+random.seed(int(sys.argv[1]))
+
+#Read the second... arguments and evaluate them as python. Example call:
+#example.py {seed} list(range(3,6)) + [2**i for i in range(5)]
+# =>               [3, 4, 5]        + [1, 2, 4, 8, 16]
+list = eval(' '.join(sys.argv[2:]).encode('ascii').decode('unicode_escape'))
```

Comment:
I like the idea of having a canonical Python generator, but I'm not sure if we should advocate the use of `eval` here :stuck_out_tongue: I'm also fine with a generator that simply takes the remaining varargs passed to the generator and shuffles them, for instance:

```py
list = list(map(int, sys.argv[2:]))
```

Also, we probably shouldn't assign to `list`, since that's a built-in function in Python :stuck_out_tongue: (perhaps `values` or `numbers`?)

Reply:
Hmm, I kinda like the `eval` suggestion though. For problems with small input, being able to write `2**10-1 10**6` is really useful and maybe not something users realize themselves. (In that case maybe rename to `eval.py`?)

But again this also doesn't matter too much.

Reply:
I agree that it can be nice, but again, I don't like having it in the default skeleton because `eval`-ing arbitrary user input isn't really the best idea. If someone likes to use this for their own problems, they can quickly add such a generator themselves (similar to the symlink from `input_validator` to `generator`) :slightly_smiling_face:

Reply:
I am not sure if `eval`-ing makes it much worse when you already execute random code inside the generator? :p

Reply:
That's fair :joy:

Either which one we choose, this generator mostly serves as an example (hence the name `example.py`) and is not intended for production use. Its fate may end up being the same as `stdout.py`, which is, being removed or unused for 90% of the problems...